container.te

```diff
@@ -31,6 +31,66 @@ gen_tunable(container_manage_cgroup, false)
 ## </desc>
 gen_tunable(container_use_cephfs, false)
 
+ifndef(`distro_ubuntu',`
```

Comment:
We should use "ifdef" instead of "ifndef". ifdef says if distro_ubuntu is defined.

Reply:
Done